src/test/java/pl/winiarczyk/lesson4/homework/MinToSecConversion/MinutesToSecondsConversionTest.java

```diff
@@ -0,0 +1,28 @@
+package pl.winiarczyk.lesson4.homework.MinToSecConversion;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MinutesToSecondsConversionTest {
+
+    double timeInMinutes;
```

Comment:
nie widzę jakiegoś większego sensu, aby to deklarować poza metodami. Rozumiem, że zrobiłeś to, bo w każdej z tych klas deklarowałbyś tę zmienną, ale patrząc w ten sposób to również
`double timeInSeconds` powinno sie tutaj znaleźć. A to powoduje trochę chaos, bo w teście nie widać typów. 

![image](https://user-images.githubusercontent.com/17989854/131352299-177ef445-ae51-435d-bbe1-acccc4a6f6b0.png)